src/pymodaq/utils/managers/parameter_manager.py

```diff
@@ -29,22 +26,30 @@ def __init__(self):
 
         self.tree.setMinimumWidth(150)
         self.tree.setMinimumHeight(300)
+        
+        self.setup_actions(action_list) # Making the button
+        if len(action_list) != 0: # If no action allowed, no need for splitter
+            self.splitter = QtWidgets.QSplitter(QtCore.Qt.Vertical)
+            self.widget.layout().addWidget(self.splitter)
+            self.widget.layout().setContentsMargins(0, 0, 0, 0)            
+            self.splitter.addWidget(toolbar)
+            self.splitter.addWidget(self.tree)
+            self.splitter.setSizes([0, 300])
+        else:
+            self.widget.layout().addWidget(self.tree)
 
-        self.splitter.addWidget(toolbar)
-        self.splitter.addWidget(self.tree)
-
-        self.splitter.setSizes([0, 300])
-        self.setup_actions()
-
-    def setup_actions(self):
+    def setup_actions(self,action_list:list = []):
         """
-
         See Also
         --------
         ActionManager.add_action
         """
-        self.add_action('save_settings', 'Save Settings', 'saveTree', "Save Settings")
-        self.add_action('load_settings', 'Load Settings', 'openTree', "Load Settings")
+        self.add_action('save_settings', 'Save Settings', 'saveTree', "Save current settings in an xml file",
+                        visible='save' in action_list)
+        self.add_action('update_settings', 'Update Settings', 'updateTree', "Update the settings from an xml file, the settings structure loaded must be identical to the current one",
```

Comment:
I think the line is a bit long and should exceed the authorized length just before "Update the settings ... enter a new line